terraform/deploy/modules.tf

```diff
@@ -172,12 +172,22 @@ module "vpc" {
   vpc_endpoint_source_sg_ids  = [module.concourse_web.outputs.security_group.id, module.concourse_worker.outputs.security_group.id]
 }
 
-module "waf" {
-  source = "../modules/waf"
+module "concourse_waf_log_group" {
+  source = "../modules/cloudwatch_log_group"
 
   name = var.name
+  tags = local.tags
 
+  group_name        = "waf"
+  retention_in_days = 30
```

Comment:
should probably be 180
> For the Management environment: all management logs should be retained for 6 months.

Reply:
Agree. This should be 180.

Reply:
Does the Analytics repo need updating to 180 as well?